tests/pages/components/task-details.js

```diff
@@ -22,7 +22,10 @@ export default {
     hasStrongText: isVisible('strong'),
     text: text()
   },
-
+  nullCommentBody:{
+    scope: '.markdown-body--empty',
```

Comment:
You'll want to change this and the other `scope` to `[data-test-markdown-body-empty]`, and you can remove the `isVisible` and call `isVisible()` directly in the test without issue.